store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9840,5 +9840,8 @@ TODO: delete them permanently from here
 	<defaultExternalCOSValue>FALSE</defaultExternalCOSValue>
 	<desc>This attribute is used to enable/disable user setting options under preference</desc>
 </attr>
+<attr id="5029" name="davuniqueid" type="string" cardinality="single" optionalIn="account" flags="accountInfo" since="8.8.12">
+	<desc>This attribute is used to track last operation performed on user</desc>
```

Comment:
This description is very vague.  Because of the "dav" prefix, I assumed this was something to do with WebDAV/CalDAV/CardDAV.  Please improve it to make it clear who the actor is.